src/harness/sas_testcase.py

```diff
@@ -167,6 +167,23 @@ def assertChannelsContainFrequencyRange(self, channels, frequency_range):
     self.assertEqual(channels[0]['frequencyRange']['highFrequency'],\
              frequency_range['highFrequency'])
 
+  def assertChannelIncludedInFrequencyRanges(self, channel, frequency_ranges):
+    """Check if the channel lies within the list of frequency ranges.
+
+    Args:
+      channel: A dictionary containing frequencyRange,
+               which is a dictionary containing lowFrequency and highFrequency
+      frequency_ranges: A list of dictionaries containing
+                        lowFrequency and highFrequency
+    """
+    isFrequencyIncludedInRange = False
```

Comment:
`is_channel_included_in_range` (both to match the name `channel` and to use `snake_case` which is our convention).

Reply:
Fixed the case to snake_case.